common/lib/xmodule/xmodule/js/src/video/06_video_progress_slider.js

```diff
@@ -89,7 +89,8 @@ function () {
                 .slider.slider('option', 'value')),
             'aria-valuemax': state.videoPlayer.duration(),
             'aria-valuemin': '0',
-            'aria-valuenow': state.videoPlayer.currentTime
+            'aria-valuenow': state.videoPlayer.currentTime,
+            'tabindex': '0'
```

Comment:
Can we change the word anchor to a `div` to avoid confusion. https://github.com/edx/edx-platform/pull/12430/files#diff-73e8105d41d0b0663d49a4d0f83d7376R82